fields/uabb-gradient/uabb-gradient.php

```diff
@@ -1,107 +1,113 @@
-<?php 
-/*
-	Declaration of array
-	
-	Ex.	'YOUR_VARIABLE_NAME'     => array(
-                        'type'          => 'uabb-gradient',
-                        'label'         => __( 'Gradient', 'uabb' ),
-                        'default'       => array			//Required NULL or Default value
-                            'color_one'		=> '',
-                            'color_two'     => '',
-                            'angle'      	=> '0',
-                        )
-                    )
+<?php
+/**
+ *  UABB Gradient file
+ *
+ *  @package UABB Gradient
+ */
 
-Note : Default value is required here. Either pass it NULL or enter your own value.
```

Comment:
show this note in the doc comments of the uabb-gradient function

Reply:
Added